### DISCUSSION TITLE
Chapter 13 - Evolutionary Games

### DISCUSSION THREAD

**Discussion by argunaykut** [softmatterlab/SOCS#22] [General]
Here, you can ask questions and discuss issues about the simulations of Chapter 13 - Evolutionary Games. 
We encourage you to participate in these discussions and engage with our community.

**Comment by amuricys** [comment]
Can someone explain what the book means with these numbers?
<img width="347" alt="Screenshot 2021-12-03 at 19 10 14" src="https://user-images.githubusercontent.com/9143614/144651852-86396025-9d23-42bd-b39e-2ca1c8f8c512.png">
 Shouldn't the current cell's "score" really be four numbers, one for each series of games played with each of its neighbors?

**Reply by acallegar** [reply]
The number at the center of each square represents the total score of the player in that square.
More in detail: the player in the center (score "27") plays against its 4 neighbours according to the rules.
The sum of all the 4 games gives 27.
Also each one of the four neighbours plays (each against each own's neighbours, so, some are not represented in the figure). It happens that each of the neighbours have its own score (indicated in the figure).

So: the figure does not represent the individual games of the central player against  its neighbours, but the final score of each player AFTER 4 games (where, for the 4 neighbours of the central playes, not all neighbours are represented).
